compiler/dyno/include/chpl/resolution/scope-types.h

```diff
@@ -232,8 +234,17 @@ class Scope {
       represents. An empty ID indicates that this Scope is the root scope. */
   const ID& id() const { return id_; }
 
-  /** Returns 'true' if this Scope directly contains use or import statements */
-  bool containsUseImport() const { return containsUseImport_; }
+  /** Returns 'true' if this Scope directly contains use or import statements
+      including the automatic 'use' for the standard library. */
+  bool containsUseImport() const {
+    return containsUseImport_ || autoUsesModules_;
+  }
+
+  /** Returns 'true' if the Scope includes the automatic 'use' for
+      the standard library. */
+  bool autoUsesModules() const {
```

Comment:
At first that might seem better, but then it would imply that it "automatically uses all standard modules" which is not the case. What I meant this query to be was  "does it automatically use any modules" where it is the case that if it automatically uses any modules it is always the same modules (right now ChapelStandard, but we are looking to change it to ChapelStandard and a few others). I could change it to `autoUsesChapelStandard` but that leaves out standard library stuff. And `autoUsesAutomaticModules` seems too repetitive to be useful. So for now I think I will leave it this way but we can adjust it if we come to a name we both like better.